unpackaged/config/rd2_post_config/flexipages/NPSP_Recurring_Donation.flexipage

```diff
@@ -0,0 +1,113 @@
+        </componentInstances>
+        <name>facet-1ac173b3-bbf9-42df-b2cf-d9c3e4adb6dc</name>
+        <type>Facet</type>
+    </flexiPageRegions>
+    <flexiPageRegions>
+        <componentInstances>
+            <componentInstanceProperties>
+                <name>relatedListComponentOverride</name>
+                <value>NONE</value>
+            </componentInstanceProperties>
+            <componentInstanceProperties>
+                <name>rowsToDisplay</name>
+                <value>10</value>
+            </componentInstanceProperties>
+            <componentInstanceProperties>
+                <name>showActionBar</name>
+                <value>true</value>
+            </componentInstanceProperties>
+            <componentName>force:relatedListContainer</componentName>
+        </componentInstances>
+        <name>facet-df729d2c-0f85-4d90-ac8b-f44fa7f5078b</name>
+        <type>Facet</type>
+    </flexiPageRegions>
+    <flexiPageRegions>
+        <componentInstances>
+            <componentInstanceProperties>
+                <name>body</name>
+                <value>facet-1ac173b3-bbf9-42df-b2cf-d9c3e4adb6dc</value>
+            </componentInstanceProperties>
+            <componentInstanceProperties>
+                <name>title</name>
+                <value>Standard.Tab.detail</value>
+            </componentInstanceProperties>
+            <componentName>flexipage:tab</componentName>
+        </componentInstances>
+        <componentInstances>
+            <componentInstanceProperties>
+                <name>active</name>
+                <value>false</value>
+            </componentInstanceProperties>
+            <componentInstanceProperties>
+                <name>body</name>
+                <value>facet-df729d2c-0f85-4d90-ac8b-f44fa7f5078b</value>
+            </componentInstanceProperties>
+            <componentInstanceProperties>
+                <name>title</name>
+                <value>Standard.Tab.relatedLists</value>
+            </componentInstanceProperties>
+            <componentName>flexipage:tab</componentName>
+        </componentInstances>
+        <name>facet-7003451d-d97d-44c2-b74f-b56f89537a36</name>
+        <type>Facet</type>
+    </flexiPageRegions>
+    <flexiPageRegions>
+        <componentInstances>
+            <componentInstanceProperties>
+                <name>tabs</name>
+                <value>facet-7003451d-d97d-44c2-b74f-b56f89537a36</value>
+            </componentInstanceProperties>
+            <componentName>flexipage:tabset</componentName>
+        </componentInstances>
+        <name>main</name>
+        <type>Region</type>
+    </flexiPageRegions>
+    <flexiPageRegions>
+        <componentInstances>
+            <componentInstanceProperties>
+                <name>displayNum</name>
+                <value>12</value>
+            </componentInstanceProperties>
+            <componentName>%%%NAMESPACE_OR_C%%%rdScheduleVisualizer</componentName>
```

Comment:
```suggestion
            <componentName>%%%NAMESPACE_OR_C%%%:rdScheduleVisualizer</componentName>
```

It turned out to be a _very_ confusing error message. The token doesn't insert the `:`, but the error message _adds_ a `c:`. Hence, we tried to deploy `crdScheduleVisualizer` and the system told us `c:crdScheduleVisualizer` didn't exist.

Reply:
Thanks @davidmreed. Follow-up question. How do I test this against a beta org? I'm trying the following to force the namespace injection, but nothing works. Tried using `-o namespaced_org true` as well.
`cci task run deploy_rd2_config --org rd2beta -o namespace_inject npsp`

Reply:
@force2b  You can safely omit `namespace_inject`, because it's specced in the task definition. For beta, use `-o unmanaged False`.

Reply:
That did it. Thank you!! I confirm that adding the `:` worked.

Reply:
Works great!!!